recipes/kaolin/LICENSE

```diff
@@ -0,0 +1,174 @@
+                                Apache License
```

Comment:
No need to include this here. The recipe will already pick it up [from the repo]( https://github.com/NVIDIAGameWorks/kaolin/blob/master/LICENSE ).